ui/components/subscription/TeamListing.tsx

```diff
@@ -217,7 +228,7 @@ export default function TeamListing({
           <BuyTeamListingModal
             selectedChain={selectedChain}
             listing={listing}
-            recipient={teamSplitAddress}
+            recipient={teamData.multisigAddress}
```

Comment:
Is this all there is to it? Do we also need to get rid of the "disperse" function on the profile page?